srcpy/bindings.cpp

```diff
@@ -203,19 +203,26 @@ BOOST_PYTHON_MODULE(libodri_control_interface_pywrap)
         .value("negative", CalibrationMethod::NEGATIVE)
         .value("alternative", CalibrationMethod::ALTERNATIVE);
 
+    bool (Robot::*RunCalibration0)() = &Robot::RunCalibration;
```

Comment:
super nice trick! this makes the code more elegant below

Reply:
Thanks the boost python doc for it :D